### PR TITLE
✨ Added Kryptonia

### PR DESCRIPTION
**Pull request by iAmaury**
Added presence for kryptonia.fr, a French minecraft server.

https://i.imgur.com/yqkNyqc.png
https://i.imgur.com/gnwVs7C.png

### CODE REVIEW

File: websites/K/Kryptonia/presence.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+    clientId: "821810069733376022"
+  });
+  presence.on("UpdateData", async () => {
+    const data: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: Date.now()
```
Comment (SlowLife1661) [inline websites/K/Kryptonia/presence.ts:7]: It's second not millisecond

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by SlowLife1661** [comment]
Also fix linebreaks

File: websites/K/Kryptonia/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Kryptonia",
+    "author": {
+      "name": "Amaury",
+      "id": "239487545791086592"
+    },
+    "category": "games",
+    "logo": "https://i.imgur.com/wbPswzO.png",
```
Comment (SlowLife1661) [inline websites/K/Kryptonia/dist/metadata.json:9]: Must at least be 512x512

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by iAmaury** [comment]
> Also fix linebreaks, don't ask whose screenshot 😸
> dontaskwhosesh

Been trying for two hours

**Comment by SlowLife1661** [comment]
> > Also fix linebreaks, don't ask whose screenshot 😸
> > dontaskwhosesh
> 
> Been trying for two hours

Maybe use Prettier or something

**Comment by iAmaury** [comment]
> > > Also fix linebreaks, don't ask whose screenshot 😸
> > > dontaskwhosesh
> > 
> > 
> > Been trying for two hours
> 
> Maybe use Prettier or something

Seems like it worked this time

File: websites/K/Kryptonia/presence.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+    clientId: "821810069733376022"
+  });
+  presence.on("UpdateData", async () => {
+    const data: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: Date.now()/1000
```
Comment (SlowLife1661) [inline websites/K/Kryptonia/presence.ts:7]: Math.floor it also

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]

**Comment by iAmaury** [comment]
It's broken, I just added three lines, there was no issue with CRLF to LF before the most recent commit, and now it's filled with that issue, but only three lines changed when checking the commit

File: websites/K/Kryptonia/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      "kryptonia",
+      "minecraft"
+    ],
+    "url": [
+      "kryptonia.fr",
+      "forum.kryptonia.fr"
+    ],
+    "description": {
+      "fr": "Kryptonia est un serveur Minecraft PvP Factions 1.7.10 sous launcher ayant ouvert ses portes en octobre 2016. Après plus d'un an d'absence, le serveur revient suite à la demande des joueurs. La communauté a fait part de ses ajouts préférés durant les six versions et les développeurs se sont afférés afin de ne garder que le meilleur, toute version confondue ! N'attendez plus et rejoignez-nous afin de redécouvrir des expériences de jeu oubliées.",
+      "en": "Kryptonia is a French Minecraft PVP Factions server."
```
Comment (Bas950) [inline websites/K/Kryptonia/dist/metadata.json:23]: the descriptions dont seem to match

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]
LGTM!
@SlowLife1661 minor things to you, cool that you are pointing out things but we are there and we don't really need any other pairs of hands to help. Another thing is CLRF or LF - it doesn't matter, just codacy checker for no reason wants everything in LF, it isn't big deal.

**Pull request merged**